src/compose/ComposeMenu.js

```diff
@@ -25,6 +25,7 @@ type Props = $ReadOnly<{|
   dispatch: Dispatch,
   expanded: boolean,
   destinationNarrow: Narrow,
+  insertAttachment: ([DocumentPickerResponse]) => Promise<void>,
```

Comment:
> `[DocumentPickerResponse]`

This doesn't mean what you think it does :smile: -- the syntax you want is `DocumentPickerResponse[]`.

(Or `$ReadOnlyArray<DocumentPickerResponse>` -- you'll see that one a lot in our code, and it's usually more exactly what we mean, but I think here the difference doesn't matter at all so we may as well do the version with the nice `[]` syntax.)

The syntax `[foo]` in Flow means an array of length 1, where the element has type `foo`. The feature is more interesting when there are several elements -- you can specify different types for each one. See docs:
https://flow.org/en/docs/types/tuples/

For array types, including `$ReadOnlyArray`, see docs here:
https://flow.org/en/docs/types/arrays/

Reply:
... Hmm, and I see that perhaps the root cause of this is that our libdef for this `react-native-document-picker` library -- where we wrote down what the types are in its interface -- has it wrong in the same way! That's the file `flow-typed/react-native-document-picker_v3.x.x.js`.

Go ahead and include a small prep commit where you fix that file :wink:

Reply:
done.